force-app/main/default/classes/NoteControllerTest.cls

```diff
@@ -0,0 +1,120 @@
+@isTest
+public with sharing class NoteControllerTest {
+    
+    @isTest
+    static void testSaveDate() {
+        
+        // create a new test note
+        Note__c testNote = new Note__C();
+        testNote.Label__c = 'Test Label';
+        testNote.Description__c = 'Test Description';
+
+        //call method SaveDate here
+        Note__c savedNote = NoteController.saveData(testNote.Label__c, testNote.Description__c);
+
+        
+
+        // Check! The recording data corresponds to the transmitted data
+        System.assertEquals(testNote.Label__c, savedNote.Label__c, 'LAbel  incorrect ');
+        System.assertEquals(testNote.Description__c, savedNote.Description__c, 'Description incorrect ');
+    }
+
+    @isTest
+    static void testDeleteNoteOnServer(){
+        
+        //create a new note for deleting
+        Note__c testNote = new Note__c();
+        testNote.Label__c = 'Test Label';
+        testNote.Description__c ='Test DEscription';
+        testNote.Created_Date__c= System.now();
+        insert testNote;
+        //call method deleteNote here
+        String result = NoteController.deleteNoteOnServer(testNote.Id);
+        //chech method return success
+        System.assertEquals('Success', result, 'Note does not delete successfully');
+
+        //check!It's really delete
+        List<Note__c> delNote = [SELECT ID FROM Note__c WHERE ID = :testNote.Id LIMIT 1 ];
+
+        System.assertEquals(0, delNote.size(), 'Note is not delete from database');
```

Comment:
Messages here is just like comment, it is not means like message for unsuccessful result.